cookiecutter.json

```diff
@@ -33,6 +34,10 @@
     "yes",
     "no"
   ],
+  "use_biber_biblatex_for_tex_bibliographies": [
```

Comment:
I think we can just remove this. Should be stable enough to always use biber/biblatex.